base/Dockerfile

```diff
@@ -36,6 +36,7 @@ RUN apk upgrade --no-cache \
     postgresql-client \
     mariadb-client \
     curl \
+    mongodb-tools
```

Comment:
This line needs to end with a \ 🤘🏼

Reply:
Can I also be pedantic and ask you to put it above curl so the database client tools are all together 😅